### PR TITLE
Added Picacoin

### PR DESCRIPTION
**Pull request by Miapata**

### CODE REVIEW

**Comment by tonymorony** [comment]
@Milerius @cipig could you please help @Miapata with Picacoin adding to Desktop GUI?

Corresponding coins repo PR: https://github.com/KomodoPlatform/coins/pull/290

**Comment by Miapata** [comment]
@tonymorony I updated everything I needed. I did the Colors in styles, I added the Picacoin logo and add the path to the logo in qml.src 😄

**Comment by Miapata** [comment]
Okay, so I saw the test and saw that it failed on the Mac build. Perhaps it was because the hex color was in lowercase. I was thinking it has something to do with the parsing of the colors. Maybe your guy's system doesn't work with lowercase hex colors? Either way, it's been updated to match the rest of the colors and makes the file look cleaner.

**Comment by cipig** [comment]
https://github.com/KomodoPlatform/coins/pull/290 does not contain the most important part, the coin config in file https://github.com/KomodoPlatform/coins/blob/master/coins

**Comment by tonymorony** [comment]
> [KomodoPlatform/coins#290](https://github.com/KomodoPlatform/coins/pull/290) does not contain the most important part, the coin config in file https://github.com/KomodoPlatform/coins/blob/master/coins

oh, right! missed that somehow

@Miapata could you please PR into https://github.com/KomodoPlatform/coins/blob/master/coins file configuration you used for the successful swap? (https://github.com/KomodoPlatform/coins/blob/master/swaps/PIC-KMD.md)

**Comment by Miapata** [comment]
@tonymorony I added it in. https://github.com/KomodoPlatform/coins/pull/295/commits

**Comment by Miapata** [comment]
@tonymorony Is there anything else needed to tet this again?

File: qml.qrc [hunk 1]
```diff
@@ -154,6 +154,7 @@
         <file>atomic_defi_design/assets/images/coins/pangea.png</file>
         <file>atomic_defi_design/assets/images/coins/pax.png</file>
         <file>atomic_defi_design/assets/images/coins/paxg.png</file>
+         <file>atomic_defi_design/assets/images/coins/pic.png</file>
```
Comment (Milerius) [inline qml.qrc:157]: coding style

Comment (Miapata) [reply qml.qrc]: Alright this is all done 😊

**Review by cipig — APPROVED** [review]

**Review by tonymorony — APPROVED** [review]

**Pull request merged**